### ISSUE TITLE
JSON errors

### ISSUE THREAD

**Issue by diedrichg** [patrickelam/mealie_cookbook_generator#1]
Unraid 6.12.13

Using the Unraid command line I was successful in cloning the repository and building the Docker container. While still in the folder where the project was cloned I created the config.ini file with the proper info and api. I then ran the 

`docker run --rm -it -v $(pwd):/usr/src/app cookbook_generator python ./pdf_generator.py --find_title_issues`

per the example usage. I was immediately presented with:

> Traceback (most recent call last):
>   File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 974, in json
>     return complexjson.loads(self.text, **kwargs)
>   File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
>     return _default_decoder.decode(s)
>   File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
>   File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
>     raise JSONDecodeError("Expecting value", s, err.value) from None
> json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/src/app/./pdf_generator.py", line 691, in <module>
>     globalRecipeCache = fetchAllRecipesWithData()
>   File "/usr/src/app/./pdf_generator.py", line 145, in fetchAllRecipesWithData
>     listData = response.json()
>   File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 978, in json
>     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
> requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

running `docker run --rm -it -v $(pwd):/usr/src/app cookbook_generator python ./pdf_generator.py -t JRG -c soups` resulted in:

> 
> Building caches...
> Traceback (most recent call last):
>   File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 974, in json
>     return complexjson.loads(self.text, **kwargs)
>   File "/usr/local/lib/python3.10/json/__init__.py", line 346, in loads
>     return _default_decoder.decode(s)
>   File "/usr/local/lib/python3.10/json/decoder.py", line 337, in decode
>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
>   File "/usr/local/lib/python3.10/json/decoder.py", line 355, in raw_decode
>     raise JSONDecodeError("Expecting value", s, err.value) from None
> json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/src/app/./pdf_generator.py", line 702, in <module>
>     globalRecipeCache = fetchAllRecipesWithData()
>   File "/usr/src/app/./pdf_generator.py", line 145, in fetchAllRecipesWithData
>     listData = response.json()
>   File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 978, in json
>     raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
> requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

Am I doing something wrong?

**Comment by patrickelam** [comment]
I'm unable to reproduce your errors exactly. I just tried to check out the project from scratch and found a couple issues, so I just pushed a couple of quick fixes.

However, with your issue, it looks like it's not parsing anything from the response object returned from the mealie API. I'm running the latest version of mealie and it's running fine for me. What version of mealie are you running? Is your API token and URL correct?
